src/frontend/src/pages/CreditsPage/CreditsPage.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Grid, Box, Button } from '@mui/material';
+import PageTitle from '../../layouts/PageTitle/PageTitle';
+import PageBlock from '../../layouts/PageBlock';
+import { useState } from 'react';
+
+const CreditsPage: React.FC = () => {
+  const names = ['Kevin Chen']; // This is a list of names, add yours here!
```

Comment:
Could you add little comments like this throughout the whole file explaining what all the code is/ what it's doing? I feel like just having them add a string to this list is pretty simple, so we could also use this opportunity to start  familiarizing them while typescript/react files and their structure as well

Reply:
IMO it makes the names line less special with more commented text. Also I suck at being concise apparently because there's a lot to introduce if it's their first time seeing react/typescript/javascript.